README.md

```diff
@@ -283,6 +283,27 @@ scss-lint --format=XML [scss-files...]
 </lint>
 ```
 
+### CheckStyle
+
+Outputs a CheckStyle XML with `<checkstyle>`, `<file>`, and `<error>` tags. Suitable for
+consumption by tools like [Jenkins](http://jenkins-ci.org/) with [CheckStyle plugin](https://wiki.jenkins-ci.org/display/JENKINS/Checkstyle+Plugin).
+
+```bash
+scss-lint --format=CheckStyle [scss-files...]
+```
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<checkstyle version="1.5.6">
```

Comment:
> Does the CheckStyle plugin require this attribute be present?

I do not know, but this is how the PHP_CodeSniffer works. It also puts this version number into the generated XML. Maybe this indicates that the XML which version of the Checkstyle definition compatible with. 
The version number has no any direct relation to the Jenkins Checkstyle plugin.